src/Cubie/Permutations.hs

```diff
@@ -0,0 +1,67 @@
+module Cubie.Permutations where
+
+import qualified Data.Vector.Unboxed as U
+
+{- Permutations of corner cubies -}
+cornerPermutationFront :: U.Vector Int
+cornerPermutationFront = U.fromList [0, 6, 2, 4, 1, 5, 3, 7]
```

Comment:
With these vectors, we can use the `backpermute` function from Data.Vector to get the new permutation given by the face to move.  
I envisage composing these with other functions that will take the entire cube state (of 4 vectors) and a given face to move, and apply the necessary permutations to the vectors.

The values are taken from the PDF from #1.

Reply:
I don't like this implementation so its superseded by the files above and I'll revert it